catalyst_voices/uikit_example/lib/examples/voices_navigation_example.dart

```diff
@@ -0,0 +1,177 @@
+                title: const Text('My Catalyst Proposals'),
+                onTap: () {},
+              ),
+              VoicesListTile(
+                trailing: const Icon(CatalystVoicesIcons.eye),
+                title: const Text('My Actions'),
+                onTap: () {},
+              ),
+              VoicesListTile(
+                trailing: const Icon(CatalystVoicesIcons.eye),
+                title: const Text('Catalyst Campaign Timeline'),
+                onTap: () {},
+              ),
+            ],
+          ),
+          const Divider(),
+          VoicesListTile(
+            leading: const Icon(CatalystVoicesIcons.user),
+            trailing: const Icon(CatalystVoicesIcons.eye),
+            title: const Text('Catalyst Roles'),
+            onTap: () => Navigator.pop(context),
+          ),
+          VoicesListTile(
+            leading: const Icon(CatalystVoicesIcons.annotation),
+            trailing: const Icon(CatalystVoicesIcons.eye),
+            title: const Text('Feedback'),
+            onTap: () => Navigator.pop(context),
+          ),
+          const Divider(),
+          VoicesListTile(
+            leading: const Icon(CatalystVoicesIcons.arrow_right),
+            trailing: const Icon(CatalystVoicesIcons.eye),
+            title: const Text('Catalyst Gitbook documentation'),
+            onTap: () => Navigator.pop(context),
+          ),
+          VoicesListTile(
+            leading: const Icon(CatalystVoicesIcons.arrow_right),
+            trailing: const Icon(CatalystVoicesIcons.eye),
+            title: const Text('Opportunity board'),
+            onTap: () => Navigator.pop(context),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _DrawerChooser extends StatefulWidget {
+  const _DrawerChooser();
+
+  @override
+  State<_DrawerChooser> createState() => _DrawerChooserState();
+}
+
+class _DrawerChooserState extends State<_DrawerChooser> {
+  _Space _selectedSpace = _Space.ideas;
+
+  @override
+  Widget build(BuildContext context) {
+    return VoicesDrawerChooser<_Space>(
+      items: _Space.values,
+      selectedItem: _selectedSpace,
+      onSelected: _onSpaceSelected,
+      itemBuilder: _itemBuilder,
+      leading: IconButton(
+        icon: const Icon(CatalystVoicesIcons.all_spaces_menu, size: 20),
+        onPressed: () {},
+      ),
+    );
+  }
+
+  void _onSpaceSelected(_Space space) {
+    setState(() {
+      _selectedSpace = space;
+    });
+  }
+
+  Widget _itemBuilder({
+    required BuildContext context,
+    required _Space item,
+    required bool isSelected,
+  }) {
+    if (isSelected) {
+      return VoicesDrawerChooserItem(
+        icon: item.getIcon(),
+        foregroundColor: item.getForegroundColor(context),
+        backgroundColor: item.getBackgroundColor(context),
+      );
+    } else {
+      return const VoicesDrawerChooserItemPlaceholder();
+    }
+  }
+}
+
+enum _Space {
+  ideas,
+  discovery,
+  proposals,
+  settings;
+
+  IconData getIcon() {
```

Comment:
I noticed that a few methods have a `get` word in the name. While this naming convention is common, it is often more concise and idiomatic in Dart to omit the word `get` from the method name.

In Dart, methods that return properties or values typically do not include the “get” prefix. Instead, they use a noun or noun phrase that describes the value being returned. This aligns with Dart’s guidelines and makes the code more readable and succinct:

```dart
IconData icon() {
  // method implementation
}
```

I believe this change will improve code readability and maintain consistency with Dart’s naming conventions. 
Thank you for considering this suggestion.

Reply:
fixed 👍